https://github.com/chromium/chromium/commit/4c736420952f355f18bdc4f4ea2d16e4514fa034

--- a/content/browser/tpcd_heuristics/opener_heuristic_utils.h
+++ b/content/browser/tpcd_heuristics/opener_heuristic_utils.h
@@ -6,6 +6,9 @@
 #define CONTENT_BROWSER_TPCD_HEURISTICS_OPENER_HEURISTIC_UTILS_H_
 
 #include <map>
+#include <set>
+#include <string>
+#include <utility>
 
 #include "base/types/optional_ref.h"
 #include "content/common/content_export.h"
